src/utils.js

```diff
@@ -0,0 +1,35 @@
+const SIGN_INPUT_ERROR_CLASSNAME = "sign-input-error";
+const ERROR_MESSAGE_CLASSNAME = "error-message-on";
+const EMAIL_REGEX = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/g;
+
+export function setInputError(elements, message) {
+  elements.input.className += ` ${SIGN_INPUT_ERROR_CLASSNAME}`;
+  elements.errorMessage.className += ` ${ERROR_MESSAGE_CLASSNAME}`;
```

Comment:
이렇게 작성하는 게 좀 더 직관적일 것 같네요!

```suggestion
elements.input.classList.add(SIGN_INPUT_ERROR_CLASSNAME);
```